### ISSUE TITLE
Fresh build in VS2013 with nuget fails on parser

### ISSUE THREAD

**Issue by nikswamy** [FStarLang/FStar#73]
I thought this issue was fixed at some point, but I just did a fresh git clone and built from scratch. Here's what I got. Antoine ... you're the nuget expert. Any idea what's going on here? I have to build parse.fs and lex.fs using the Makefile the first time and then everything works out.

```
------ Build started: Project: parser, Configuration: Debug Any CPU ------
	"C:\Users\nswamy\workspace\FStar\VS\.nuget\NuGet.exe" install "C:\Users\nswamy\workspace\FStar\src\parser\packages.config" -source ""  -NonInteractive -RequireConsent -solutionDir "C:\Users\nswamy\workspace\FStar\VS\ "
	Restoring NuGet packages...
	To prevent NuGet from downloading packages during build, open the Visual Studio Options dialog, click on the Package Manager node and uncheck 'Allow NuGet to download missing packages'.
	Installing 'FsLexYacc 6.0.3'.
	Installing 'FsLexYacc.Runtime 6.0.2'.
	Successfully installed 'FsLexYacc.Runtime 6.0.2'.
	Successfully installed 'FsLexYacc 6.0.3'.
	C:\Program Files (x86)\Microsoft SDKs\F#\3.1\Framework\v4.0\fsc.exe -o:obj\Debug\parser.dll -g --debug:full --noframework --define:TRACE --define:DEBUG --optimize+ -r:C:\Users\nswamy\workspace\FStar\bin\absyn.dll -r:C:\Users\nswamy\workspace\FStar\bin\basic.dll -r:C:\Users\nswamy\workspace\FStar\bin\FSharp.PowerPack.Compatibility.dll -r:C:\Users\nswamy\workspace\FStar\bin\FSharp.PowerPack.dll -r:C:\Users\nswamy\workspace\FStar\VS\packages\FsLexYacc.Runtime.6.0.2\lib\net40\FsLexYacc.Runtime.dll -r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\mscorlib.dll" -r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\System.Core.dll" -r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\System.dll" -r:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\System.Numerics.dll" --target:library --warn:1 --warnaserror:76 --vserrors --validate-type-providers --LCID:1033 --utf8output --fullpaths --flaterrors --subsystemversion:6.00 --highentropyva+ --sqmsessionguid:b3e40ddb-dfc0-46dd-9640-e8e9ddf9283b --mlcompatibility "C:\Users\nswamy\AppData\Local\Temp\.NETFramework,Version=v4.5.AssemblyAttributes.fs" ast.fs parseutil.fs parse.fs lexhelp.fs lex.fs dsenv.fsi dsenv.fs desugar.fsi desugar.fs parseit.fsi parseit.fs driver.fs 
FSC: error FS0225: Source file 'C:\Users\nswamy\workspace\FStar\src\parser\parse.fs' could not be found
FSC: error FS0225: Source file 'C:\Users\nswamy\workspace\FStar\src\parser\lex.fs' could not be found
Done building project "parser.fsproj" -- FAILED.

Build FAILED.
```

**Comment by nikswamy** [comment]
Is there any way to resolve this issue? What's wrong with shipping FsLex/FsYacc build rules. 

Yesterday, someone was complaining about the VS build not working out of the box.

**Comment by gsvgit** [comment]
In order to automate VisualFStar build with appveyor we create build.cmd script for build FStar out of the box. You can found  it here: https://github.com/YaccConstructor/VisualFStar . It may be useful for other developers.

**Comment by nrolland** [comment]
hi, I am having a similar bug elsewhere. @ad-l would you be able to precise what you mean by 

```
Yes, as I said before this is because of the build rules added by FsLexYacc that can
 ***only be loaded together with the project file***. 

***Reloading the parser project fixes it*** but if we want it to work from scratch we need
 to ship the file that defines the FsLex/FsYacc build rules.```

**Comment by mlr-msft** [comment]
Should now be addressed in duplicate issue #430.
